tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -365,6 +365,7 @@ public function theseUsersHaveBeenCreatedWithoutSkeletonFiles(TableNode $table)
 
 	/**
 	 * @Given /^these users have been created with ?(default attributes and|) skeleton files ?(but not initialized|):$/
+	 * @When /^the administrator creates a regular user with ?(default attributes and|) skeleton files ?(but not initialized|):$/
```

Comment:
```suggestion
	 * @When /^the administrator creates a these users with ?(default attributes and|) skeleton files ?(but not initialized|):$/
```
At the moment it sounds like no table is required. But actually this step has to have a table of users, and it can have 1 or many users that can have any user names.